requirements.txt

```diff
@@ -1,12 +1,87 @@
+alabaster==0.7.9
+ansicolor==0.2.4
+appdirs==1.4.0
+astroid==1.4.9
+autoflake==0.6.6
+autopep8==1.2.4
+Babel==2.3.4
+bandit==1.4.0
+chardet==2.3.0
+click==6.7
+cmakelint==1.3.4.1
+coala==0.9.1
+coala-bears==0.9.2
+coala-utils==0.5.1
+colorama==0.3.7
+colorlog==2.10.0
+cppclean==0.12
+cpplint==1.3.0
+decorator==4.0.11
 defusedxml==0.4.1
 Django==1.8
+django-ckeditor==5.1.1
+django-email-extras==0.3.3
+django-forms-builder==0.13.0
+docopt==0.6.2
+docutils==0.13.1
+eradicate==0.1.6
+future==0.15.0
+gitdb2==2.0.0
+GitPython==2.1.1
+guess-language-spirit==0.5.2
+html-linter==0.3.0
+imagesize==0.7.1
+ipython-genutils==0.1.0
+isort==4.2.5
+Jinja2==2.9.4
+jsonschema==2.5.1
+jupyter-core==4.2.1
+lazy-object-proxy==1.2.2
+libclang-py3==0.2
+mando==0.3.3
+MarkupSafe==0.23
+mccabe==0.6.0
+munkres3==1.0.5.5
+mypy-lang==0.4.6
+nbformat==4.2.0
+nltk==3.2.2
 oauthlib==1.1.2
+packaging==16.8
+pbr==1.10.0
+pep8==1.7.0
 Pillow==3.3.1
+proselint==0.7.0
+pydocstyle==1.1.1
+pyflakes==1.5.0
+Pygments==2.2.0
 PyJWT==1.4.2
+pylint==1.6.5
+pyparsing==2.1.10
+PyPrint==0.2.6
+python-gnupg==0.3.9
 python-social-auth==0.2.21
 python3-openid==3.0.10
-requests==2.11.1
+pytz==2016.10
+PyYAML==3.12
+radon==1.4.0
+requests==2.13.0
 requests-oauthlib==0.6.2
+restructuredtext-lint==0.17.2
+rstcheck==2.2
+safety==0.5.1
+scspell3k==2.1
 six==1.10.0
-django-ckeditor==5.1.1
-django-forms-builder==0.13.0
+smmap2==2.0.1
```

Comment:
You don't need all these requirements, you could just use the `cib` and `coala` package
and run the script `cib install -c .coafile` @ghoshbishakh